pages/windows/choco-upgrade.md

```diff
@@ -0,0 +1,31 @@
+# choco upgrade
+
+> Upgrade one or more packages with Chocolatey.
+
+- Upgrade one or more space-separated packages:
+
+`choco upgrade {{package(s)}}`
+
+- Upgrade to a specific version of a package:
+
+`choco upgrade {{package}} --version {{version}}`
+
+- Upgrade all packages:
+
+`choco upgrade all`
+
+- Upgrade all except specified comma-seperated packages:
```

Comment:
typo at "seperated"

Reply:
:man_facepalming: I make that mistake so often. Cheers for pointing it out.